### PR TITLE
orientation feature for linux

### PR DESCRIPTION
**Pull request by phunsukwangdu**
Orientation feature for linux. It has been implemented with the help of xrandr.
Pr is in working condition and ready to be merged.

### CODE REVIEW

File: plyer/platforms/linux/orientation.py [hunk 1]
```diff
@@ -0,0 +1,23 @@
+import subprocess as sb
+from plyer.facades import Orientation
+
+
+class AndroidOrientation(Orientation):
```
Comment (malverick) [inline plyer/platforms/linux/orientation.py]: I think it should be LinuxOrientation

Comment (malverick) [inline plyer/platforms/linux/orientation.py]: I think it should be LinuxOrientation

File: plyer/platforms/linux/orientation.py [hunk 2]
```diff
@@ -0,0 +1,23 @@
...
+    def _set_portrait(self, **kwargs):
+        self.rotate = 'left'
+        self.screen = sb.check_output("xrandr -q | grep ' connected' \
+                              |  head -n 1 | cut -d ' ' -f1", shell=True)
+        self.screen = self.screen.split('\n')[0]
+        sb.call(["xrandr", "--output", self.screen, "--rotate", self.rotate])
+
+
+def instance():
+    return AndroidOrientation()
```
Comment (malverick) [inline plyer/platforms/linux/orientation.py]: return LinuxOrientation()

**Comment by phunsukwangdu** [comment]
thx malverick i was doing the changes meanwhile you spotted it. :)

**Comment by akshayaurora** [comment]
@phunsukwangdu thanks 👍 could you add a try/except and throw a appropriate error just to exit cleanly when Xrandr is not installed.

**Comment by bhaveshAn** [comment]
Using `shell=True` is hazardous Please see https://docs.python.org/2/library/subprocess.html#frequently-used-arguments

**Pull request merged**